modules/developers-guide/pages/data_federation.adoc

```diff
@@ -0,0 +1,87 @@
+their own data sources, business logic, and corresponding microservices.
+In an unfederated implementation, we would have this simple Schema and Resolvers
+owned and implemented by the Studio API team.
+The GraphQL Framework would take in queries from clients and orchestrate the
+calls to the resolvers in a breadth-first traversal.
+
+SEE DIAGRAM
+
+This ability to extend a Movie type across GraphQL service boundaries makes Movie
+a Federated Type.
+Resolving a given field requires delegation by a gateway layer down to the
+owning domain services.
+
+
+****
+CQL indexes can be created on regular (not partition key nor clustering columns) fields.
+Parameters
+Queries are parameterized by:
+partition key and clustering fields
+regular fields, if they are indexed
+
+Stargate only allows combinations of fields that won't require an ALLOW FILTERING
+at the CQL level:
+- at most one indexed field can be present in the signature. If so, it must
+either be the only parameter, or the other parameters must cover the full
+partition key (no clustering columns are allowed).
+- if no indexed field is present, then all partition key fields must be present,
+but it's possible to omit some or all of the clustering fields (in which case
+  the query will return multiple rows).
+
+The arguments must have the same names and types as the fields on the object type.
+They don't necessarily have to be in order (although it's probably the most
+logical to keep the same order).
+****
+
+For the Library example:
+
+*** 1-4 are NOT POSSIBLE ***
+1. (allBooks) Get all books. _Book_
+2. (allReaders) Get all readers. _Reader_
+3. (allOrders) Get all checkout orders. _Order_
+4. (allLibColls) Get all lib collections. _LibCollection_
+
+5. (booksByTitle) Look up books by title (partition key). _Book_
+6. (bookIByIsbn) Look up a book by ISBN (clustering column). _BookI_
+7. (booksByTitleAndIsbn) Look up a book by title and ISBN (primary key). _Book_
+8. (booksByAuthor) Look up books by author (non-PK column collection, index). _Book_
+9. (booksByTitleAndAuthor) Look up a book by title and author (partition key + index). _Book_
+
+10. (readerByName) Look up a reader by name (partition key). _Reader_
+11. (readerUByUserid) Look up a reader by user_id (clustering column). _ReaderU_
+12. (readerByNameAndUserid) Look up a reader by name and user_id (primary key). _Reader_
+    All 3 of these queries will allow you to get all the books reviewed by a reader.
+
+13. (orders) Get all orders. _Order_
+13a. (order) Get order by checkout_id. _Order_
+13b. (orderById) Look up an order by checkout_id (partition key). _Order_
+    This will give you all books in the order.
+
+14. CAN'T DO?(libCollByType) Look up a library collection by type (partition key, partial primary key). _LibCollection_
+15. CAN'T DO(libCollByLibid) Look up a library collection by lib_id (partition key, partial primary key). _LibCollection_
```

Comment:
Is there an answer yet on whether you can or can't do this?

Reply:
Removed page.